src/_pytest/config/__init__.py

```diff
@@ -123,7 +145,7 @@ def main(args=None, plugins=None) -> Union[int, ExitCode]:
             formatted_tb = str(exc_repr)
             for line in formatted_tb.splitlines():
                 tw.line(line.rstrip(), red=True)
-            return ExitCode.USAGE_ERROR
+            return ExitCode.INTERNAL_ERROR
```

Comment:
INTERNAL_ERROR I believe is reserved for when pytest itself crashes -- this is still a user-space error so I think the return code should stay the same here?

Reply:
In #993 was suggestion to change ``USAGE_ERROR`` into ``INTERNAL_ERROR``. But I personally agree with your reasoning.